main.cpp

```diff
@@ -0,0 +1,57 @@
+#include <iostream>
+#include <fstream>
+#include <vector>
+
+#define STB_IMAGE_WRITE_IMPLEMENTATION
+#include "stb_image_write.h"
```

Comment:
Bei uns machen wir die includes in der folgenden Reihenfolge: vom speziellen, also eigene Header zum allgemeinen, also std Bibliotheken.